### ISSUE TITLE
Add decimalPlaces (a11y) for our ProportionalPartitionLineNode

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/area-model-common#151]
Doesn't have access to it, and creating an issue so I don't forget.
